winterfell/webapp/src/js/claimStartCtrl.js

```diff
@@ -14,7 +14,9 @@ app.controller('claimStartCtrl', ['$scope', '$state', 'net', 'claimService', '$u
           return;
         }
       }
-      net.startClaim().then(function (res) {
+      net.startClaim(
+        {forms: ["VBA-21-0966-ARE"]} // hardcoded to create claim with only this form for now
```

Comment:
That sounds reasonable, basically I was thinking that somehow the set of forms should be determined by the type of claim the person chooses to file, and we would give them certain "pre bundled" options or something. I'd like to leave it for a future revision for now though I'm not sure what to do here atm